assets/css/translate.scss

```
@@ -0,0 +1,40 @@
+// Translation results of " Section x" in articles
+// Autor: Olsza
+// License: CC-BY-4.0
+
+// Enter the locale code and the translation of "Section" in the language.
+// The country code should be the same as the value of the lang attribute in html (use "-" instead of "_")
+$section_translation: (
+        "de": "",
+        "el": "",
+        "en": "",
+        "es": "",
```

```suggestion
        "en": "section",
        "es": "sección",
```

We can add English and Spanish

---

Generally I was wondering whether add EN here, because it will be like the original anyway ;)... and as for the translation of ES - super thank you